.github/workflows/ci.yaml

```diff
@@ -61,9 +61,9 @@ jobs:
       - lint
       - unit-test
       - build
-    uses: canonical/data-platform-workflows/.github/workflows/integration_test_charm.yaml@v16.2.1
+    uses: canonical/data-platform-workflows/.github/workflows/integration_test_charm.yaml@v24.0.6
     with:
-      artifact-prefix: packed-charm-cache-true
+      artifact-prefix: packed-charm-cache-false # TODO: use true after adding to ccc-hub
```

Comment:
After merging this PR, I'll add it to the cache hub and open a new PR to use the cache